apps/api/src/modules/user/user.service.ts

```diff
@@ -1,4 +1,14 @@
 import { Injectable } from '@nestjs/common'
 
+import { AuthRepository } from '../auth/auth.repository'
```

Comment:
Not sure that it is good convention or not to use service of others module (Summon @miello)

Reply:
@miello said that it is OK